build.sbt

```diff
@@ -0,0 +1,38 @@
+scalaVersion := "2.12.7" // Also supports 2.11.x
+
+val circeVersion = "0.9.3"
+val http4sVersion = "0.16.6a"
+val pac4jVersion = "3.6.1"
+val specs2Version = "3.8.9"
+
+// Only necessary for SNAPSHOT releases
+resolvers += Resolver.sonatypeRepo("snapshots")
+resolvers += "opensaml Repository" at "https://build.shibboleth.net/nexus/content/repositories/releases"
+
+libraryDependencies ++= Seq(
+  "io.circe" %% "circe-core" % circeVersion,
+  "io.circe" %% "circe-jawn" % circeVersion,
+  "org.http4s" %% "http4s-dsl" % http4sVersion,
+  "org.http4s" %% "http4s-blaze-server" % "0.16.5a",
+  "org.http4s" %% "http4s-blaze-client" % "0.16.5a",
+  "org.pac4j" % "pac4j-core" % pac4jVersion,
```

Comment:
Generally, an implementation (x-pac4j) only relies on the `pac4j-core` dependency. Only the demos putting pac4j into action and which uses CAS, SAML, OIDC, ... use the `pac4j-cas`, `pac4j-saml`, `pac4j-oidc`, ... dependencies...